src/site/content/en/blog/optimize-cls/index.md

```diff
@@ -0,0 +1,348 @@
+*   When the ad fills a container (resizes if the final ad has a different size)
+
+The good news is that it's possible for sites to follow best practices to reduce ad shift. Sites can mitigate these layout shifts by:
+
+* Statically reserving space for the ad slot
+  * In other words, style the element before the ad tag library loads
+  * If placing ads in the content flow, ensure shifts are eliminated by reserving the slot size. These ads _shouldn't_ cause layout shifts if loaded off-screen.
+* Take care when placing non-sticky ads near the top of the viewport. 
+  * In the below example, it's recommended to move the ad to below the "world vision" logo and make sure to reserve enough space for the slot.
+* Eliminate shifts by reserving the largest possible size for the ad slot
+  * Works, but risks having a blank space if a smaller ad creative fills the slot
+* Choose the most likely size for the ad slot based on historical data
+
+Some sites may find collapsing the slot initially can reduce layout shifts if the ad slot is unlikely to fill. There isn't an easy way to choose the exact size each time, unless you control the ad serving yourself.
+
+  <figure class="w-figure">
+    <video controls loop muted class="w-screenshot" poster="video-final-ads-no-dimensions.jpg">
+      <source src="https://storage.googleapis.com/web-dev-assets/optimize-cls/final_ads_no_dimensions.webm" type="video/webm;">
+      <source src="https://storage.googleapis.com/web-dev-assets/optimize-cls/final_ads_no_dimensions.mp4" type="video/mp4; codecs=h264">
+    </video>
+   <figcaption class="w-figcaption">
+      Ads without sufficient space reserved.
+    </figcaption>
+  </figure>
+
+  <figure class="w-figure">
+    <video controls loop muted class="w-screenshot" poster="video-top-ad-after.jpg">
+      <source src="https://storage.googleapis.com/web-dev-assets/optimize-cls/top-ad-after.mp4" type="video/mp4; codecs=h264">
+    </video>
+   <figcaption class="w-figcaption">
+      Ads with sufficient space reserved.
+    </figcaption>
+  </figure>
+
+<figure class="w-figure">
+  <img src="Optimize-Cumulative6.png " alt="Lighthouse report showing the before/after impact to Cumulative Layout Shift of reserving space for banners like ads">
+  <figcaption class="w-figcaption">
+    Lighthouse 6.0 impact of reserving space for this banner on CLS
+  </figcaption>
+</figure>
+
+#### Statically reserve space for the ad slot
+
+Statically style slot DOM elements with the same sizes passed to your tag library. This can help ensure the library doesn't introduce layout shifts when it loads. If you don't do this, the library may change the size of the slot element after page layout.
+
+Also consider the sizes of smaller ad serves. If a smaller ad is served, a publisher can style the (larger) container to avoid layout shifts. The downside to this approach will increase the amount of blank space, so keep in mind the trade-off here.
```

Comment:
```suggestion
Also consider the sizes of smaller ad serves. If a smaller ad is served, a publisher can style the (larger) container to avoid layout shifts. The downside to this approach is that it will increase the amount of blank space, so keep in mind the trade-off here.
```